code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -44,21 +49,21 @@
 
 /datum/species/vampire/spec_life(mob/living/carbon/human/C, delta_time, times_fired)
 	. = ..()
-	if(istype(C.loc, /obj/structure/closet/crate/coffin))
-		C.heal_overall_damage(2 * delta_time, 2 * delta_time, 0, BODYPART_ORGANIC)
-		C.adjustToxLoss(-2 * delta_time)
-		C.adjustOxyLoss(-2 * delta_time)
-		C.adjustCloneLoss(-2 * delta_time)
+	if(istype(C.loc, /obj/structure/closet)) // SKYRAT EDIT - NORMAL CLOSETS INSTEAD OF COFFINS.
+		C.heal_overall_damage(1.5 * delta_time, 1.5 * delta_time, 0, BODYPART_ORGANIC) // SKYRAT EDIT - ORIGINAL 2 - Fast, but not as fast due ot them being able to use normal lockers.
+		C.adjustToxLoss(-1 * delta_time) // SKYRAT EDIT - ORIGINAL 2 - 50% base speed to keep it fair
+		C.adjustOxyLoss(-2 * delta_time) // SKYRAT EDIT - ORIGINAL 2 - None here, oxy should heal fast
+		C.adjustCloneLoss(-0.5 * delta_time) // SKYRAT EDIT - ORIGINAL 2 - HARDMODE DAMAGE
 		return
 	C.blood_volume -= 0.125 * delta_time
 	if(C.blood_volume <= BLOOD_VOLUME_SURVIVE)
 		to_chat(C, span_danger("You ran out of blood!"))
 		var/obj/shapeshift_holder/H = locate() in C
 		if(H)
 			H.shape.dust() //make sure we're killing the bat if you are out of blood, if you don't it creates weird situations where the bat is alive but the caster is dusted.
-		C.dust()
+		C.adjustToxLoss(4)
```

Comment:
I'd probably make it hurt more than that. Considering they're out of blood and should be basically dead because of it.